test-requirements.txt

```
@@ -0,0 +1 @@
+odoo_test_helper
```

```suggestion
odoo_test_helper
```

may be not use. if you inherit CommonTierValidation

---

If there is no test on Git, it will cause an error.